doc/install_urcap.md

```diff
@@ -0,0 +1,31 @@
+# How to install the RS485 URCap
+
+This is a short step-by-step guide how to install the RS485 URCap in Polyscope.
+We assume that you copied the latest *RS485 URCap* from [here](https://github.com/UniversalRobots/Universal_Robots_ToolComm_Forwarder_URCap/releases) to the `programs` folder of your e-series robot.
```

Comment:
Linking back to the description how to copy it there might be useful.

Reply:
@fmauch Actually, that's about it with hints on how to copy. The other part in the top-level README.md just says *via scp or via usb stick*. Link back anyways?

Reply:
Yes, that's what I meant.
```suggestion
We assume that you [copied](../README.md#installation) the latest *RS485 URCap* from [here](https://github.com/UniversalRobots/Universal_Robots_ToolComm_Forwarder_URCap/releases) to the `programs` folder of your e-series robot.
```